recipe/meta.yaml

```diff
@@ -47,13 +53,13 @@ requirements:
     - requests >=2.31.0
     - scipy
     - skyfield >=1.12
-    - skyfield-data >=6
+    - skyfield-data >=7
     - tk
     - owslib >=0.24
     - unicodecsv
     - fs_filepicker >=0.3.9
     - cftime >=1.0.1
-    - matplotlib >=3.5.3
+    - matplotlib-base >=3.5.3
```

Comment:
the msui will need the qt bindings, https://mss.readthedocs.io/en/stable/tutorial.html
that is not the major issue here

Reply:
I agree that this was not the show stopper. It's just considered good practice. I don't know if people ever use mss without the gui elements, but the number of cli options suggests they do. In that case, it's better not to force every user to install the full GUI stack.

Reply:
We have only few server setups but many GUI users. Usually you need a connection to the ECMWF weather cloud, or provide your own model data by using a server. 
There you need likly access to ECMWF or some similiar service.

The server is based on OGC WMS and extended to be able to provide sideview (vertical) data and data along the flightpath. 

Common  OGC WMS clients can only show side views. We need for flight preparation, telling the pilot how we want to measure all the features.

Reply:
the other server is a socketIO server for enabling realtime planning and exchanging data.

I started a draft already to split the package for server installation. We have to split modules first. We likly will have later it similiar as basemap has done.